cg/meta/compress/files.py

```diff
@@ -154,4 +162,12 @@ def check_fastqs(compression_obj: CompressionData) -> bool:
         LOG.info("Run %s has symbolic link, skipping run", compression_obj.run_name)
         return False
 
+    date_changed = compression_obj.get_change_date(compression_obj.fastq_first)
+    delta: datetime.datetime = date_changed + FASTQ_DATETIME_DELTA
+    today = datetime.datetime.now()
+    # Check if date is older than FASTQ_DELTA
+    if delta > today:
+        LOG.info("FASTQ files are not old enough")
+        return False
+
```

Comment:
Could this logic be made more intuitive?
Rephrazing it as:
date changed < today - FASTQ_DATETIME_DELTA
I think is more readable

Reply:
Sure

Reply:
Fixed